### PR TITLE
Implement career list section

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  </div>
+</template>
+
+<script type="module">
+import CollapseTransition from "@ivanv/vue-collapse-transition/src/CollapseTransition.vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import axios from "axios";
+import config from "@/config.js";
+
+import { faApple, faAndroid } from "@fortawesome/free-brands-svg-icons";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:79]: Can't we add all icons to jobsScreen page?

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+  faPenNib,
+} from "@fortawesome/free-solid-svg-icons";
+
+export default {
+  data() {
+    return {
+      careers: [],
+      currentCareerIndex: 0,
+      isOpenCareerList: true,
+      previousCareerIndex: null,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:92]: Rename 
currentCareerIndex => currentIndex
isOpenCareerList => openList
previousCareerIndex => previousIndex

Why previousCareerIndex is null? can't we take 0?

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        </div>
+      </div>
+    </div>
+
+    <div class="career-container normal-text mt-5">
+      <div class="career-list-container">
+        <div v-for="career in careers" :key="career" class="career-list">
+          <div
+            class="career-header normal-text text-left"
+            @click="openCareer(career.id)"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:26]: openCareer => expandListItem

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+          >
+            <div class="career-icon">
+              <font-awesome-icon
+                class="gradient-career-icon"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:30]: gradient-career-icon => gradient-icon

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+
+      this.currentCareerIndex = index;
+      this.previousCareerIndex = this.currentCareerIndex;
+    },
+    getCareer() {
+      axios
+        .get(config.API_BASE + "/api/careers")
+        .then((response) => {
+          this.careers = response.data;
+          this.careers[0].icon_name = faApple;
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:118]: Why is it taken statically? Can't we use for loop here?

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+.gradient-btn > span {
+  font-size: 1.125rem;
+  line-height: 1.364rem;
+}
+
+.gradient-career-icon {
+  color: #f2709c;
+}
+
+.gradient-border-btn {
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:218]: I think we reused gradient-border-btn in the whole website. Why do we need to take it separately here? We can override the CSS of that btn if needed

File: vue-frontend/src/components/jobs/CareerView.vue [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+  .career-summary {
+    padding: 32px 48px;
+  }
+
+  .description {
+    margin: 50px 80px;
+  }
+
+  .career-container {
+    margin: 0 80px;
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/CareerView.vue:274]: Only top-bottom margin should be fix, We can take left-right margins and padding in %

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by jimmy0251 — APPROVED** [review]

**Pull request merged**